src/HealthChecks.RavenDB/DependencyInjection/RavenDBHealthCheckBuilderExtensions.cs

```diff
@@ -35,5 +35,34 @@ public static class RavenDBHealthCheckBuilderExtensions
                 sp => new RavenDBHealthCheck(connectionString, databaseName),
                 failureStatus,
                 tags));
+
+
+        /// <summary>
+        /// Add a health check for RavenDB.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHealthChecksBuilder"/>.</param>
+        /// <param name="urls">The list of URL addresses that point to RavenDB server nodes.</param>
+        /// <param name="databaseName">The specified database to check.</param>
+        /// <param name="name">
+        /// The health check name. Optional. If <see langword="null"/> the type name 'ravendb' will be used for the name.
+        /// </param>
+        /// <param name="failureStatus">
+        /// The <see cref="HealthStatus"/> that should be reported when the health check fails. Optional. If <see langword="null"/> then
+        /// the default status of <see cref="HealthStatus.Unhealthy"/> will be reported.
+        /// </param>
+        /// <param name="tags">A list of tags that can be used to filter sets of health checks. Optional.</param>
+        /// <returns>The <see cref="IHealthChecksBuilder"/>.</returns></param>
+        public static IHealthChecksBuilder AddRavenDB(
+            this IHealthChecksBuilder builder,
+            string[] urls,
```

Comment:
Could be "connectionsstrings" better name than "urls"? or "url" instead of connectionstring in the other overload!

Reply:
Hi @unaizorrilla I did consider using `connectionStrings`. However, decided aginast it since the raven documentation uses the term `url` instead of connectionString. I actually thought to rename original method parameter to `url` instead, but since it is an implementation already used by consumers did not want to change it.

Reply:
I had an additional suggestion if you like to consider it. It is to make the original method as `Obsolete` since Raven takes url array in `DocumentStore`. This way we would be consistent with the Raven documentation/implementation. However, will leave the decision on you..

Reply:
Ok!

Reply:
Great will make the relevant changes and update the PR.

Reply:
Thansk 

I try to merge asap!

Reply:
Please let me know when you update the PR

Reply:
Hi @unaizorrilla I have made some additional changes to allow user to pass raven auth certificate. To achieve this I have introduced a new parameter `RavenDBOptions` much inspired by existing code for other providers. In addition to this, the existing method which takes single `connectionString` as overload has been marked `Obsolete`